src/config.ts

```diff
@@ -439,15 +440,15 @@ const store = {
   showOnlySeries: envOrArray(process.env.SHOW_ONLY_SERIES, [
     '3060',
     '3060ti',
-    '3070',
-    '3080',
+    // '3070',
+    '3070ti',
     '3080ti',
     '3090',
     'rx6700xt',
     'rx6800',
     'rx6800xt',
     'rx6900xt',
-    'ryzen5600',
+    //'test:series'
```

Comment:
```suggestion
    'test:series'
```

Reply:
Hi Jef, thanks for the feedback. I've fixed what you said and I hope to have updated correctly the doc, I was not sure what to edit :)